openstack_sdk/common.py

```diff
@@ -56,6 +62,21 @@ def validate_resource_identifier(self):
 
         return error_message
 
+    def get_quota_sets(self):
+        project_name = self.client_config.get('project_name')
+        quota = getattr(
+            self.connection,
+            'get_{0}_quotas'.format(self.resource_type))(project_name)
+
+        if not quota:
+            raise QuotaException(
+                'Invalid {0} quota response'.format(self.resource_type))
+
+        return quota
+
+    def resource_plural(self, openstack_type):
+        return '{0}s'.format(openstack_type)
+
```

Comment:
Any reason that this can't go in the Server Group class? Is it useful for all types?

Reply:
Yeah, because this need to be used for all openstack resources, not just server group